### PR TITLE
Add method for creating a snapshot

### PR DESCRIPTION
**Pull request by andre-m-dev**
# Pull Request

## Related issue
Fixes [504](https://github.com/meilisearch/meilisearch-ruby/issues/504)

## What does this PR do?
- Add a method to create a snapshot

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

### CODE REVIEW

**Review by curquiza — COMMENTED** [review]
Thank you @andre-m-dev 
LGTM but I let one of the maintainers (@ellnix or @brunoocasali) review and merge it

File: spec/meilisearch/client/snapshots_spec.rb
Comment (brunoocasali) [inline spec/meilisearch/client/snapshots_spec.rb:5]: Be sure to test if the returned task type has the correct type: `snapshotCreation`.

File: spec/meilisearch/client/snapshots_spec.rb
Comment (brunoocasali) [inline spec/meilisearch/client/snapshots_spec.rb:11]: You don't need to check if the method was successful or not, you can remove this two lines :)

```suggestion
```

**Review by brunoocasali — CHANGES_REQUESTED** [review]
Hi! Thanks for the PR I just left some comments :)

File: spec/meilisearch/client/snapshots_spec.rb
Comment (andre-m-dev) [reply spec/meilisearch/client/snapshots_spec.rb:5]: done

**Review by ellnix — APPROVED** [review]
LGTM, thank you for the contribution @andre-m-dev 

bors merge

**Comment by ellnix** [comment]
Unfortunately it looks like we have to wait for @brunoocasali to confirm that his requested changes were addressed, I will leave merging to him.

**Comment by curquiza** [comment]
Really weird, you are supposed to have the right to merge.

Capture d’écran 2024-01-17 à 14 14 59

And you also have the right to `bors merge`, I've just checked

**Comment by curquiza** [comment]
@ellnix if the contributor made what Bruno asked for you can
- dismiss the request of change here
Capture d’écran 2024-01-17 à 14 17 15

- merge

**Comment by ellnix** [comment]
> @ellnix if the contributor made what Bruno asked for you can
> 
>     * dismiss the request of change here
> 
>     * merge

I didn't think that could be an option. Will do right away.

**Comment by ellnix** [comment]
bors merge

**Pull request merged**
